src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/MlInformationList.cs

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+
+namespace Tizen.MachineLearning.Inference
+{
+    /// <summary>
+    /// The MlInformationList class manages list of MlInformation.
+    /// </summary>
+    /// <since_tizen> 13 </since_tizen>
+    public class MlInformationList : IDisposable
+    {
+        private IntPtr _handle = IntPtr.Zero;
+        private bool _disposed = false;
+
+        /// <summary>
+        /// Creates a MlInformationList instance.
+        /// </summary>
+        /// <feature>http://tizen.org/feature/machine_learning.inference</feature>
+        /// <exception cref="NotSupportedException">Thrown when the feature is not supported.</exception>
```

Comment:
Please check the exception here and below.
I think in this case `ArgumentException`.
```suggestion
        /// <exception cref="ArgumentException">Thrown when the method failed due to an invalid parameter.</exception>
```

Reply:
Added `ArgumentException`
Thank you :)